pkg/products/bamboo/helm_release.tf

```diff
@@ -59,4 +59,34 @@ data "kubernetes_service" "bamboo" {
     name      = "bamboo"
     namespace = kubernetes_namespace.bamboo.metadata[0].name
   }
+}
+
+resource "helm_release" "bamboo_agent" {
+  name       = "bamboo-agent"
+  namespace  = kubernetes_namespace.bamboo.metadata[0].name
+  repository = "https://atlassian.github.io/data-center-helm-charts"
+  chart      = "bamboo-agent"
+  version    = "0.0.2"
```

Comment:
We should extract this into `locals` and use the same version for `bamboo` and `bamboo-agent` `helm_release` resources.

Reply:
Done.